### ISSUE TITLE
epylint.run_py does not work on Windows when exe are generated

### ISSUE THREAD

**Issue by xmedeko** [PyCQA/pylint#1324]
`epylint.py_run()` dows not work for me on Windows. The `Scripts` dir has no `*.bat` files. just `*.exe`, like `pylint.exe`, `epilint.exe`, etc.

When I remove the line
```python
script += '.bat'
```
from the `epylint.py` then it runs well.

### pylint --version output
pylint 1.6.5,
astroid 1.4.9
Python 3.5.2 (v3.5.2:4def2a2901a5, Jun 25 2016, 22:01:18) [MSC v.1900 32 bit (Intel)]

**Comment by rogalski** [comment]
Thanks for report.

In terms of fixes: maybe we can iterate over `PATHEXT` envvar, and select first matching file with extension?

**Comment by xmedeko** [comment]
I see it's already fixed in master by the commit bcd687e13de0c9ea484c9da00e2ff520abf82dac 
Probably this fix should make it to the 1.6 branch?

**Comment by rogalski** [comment]
If it's fixed in master, it'll be published in next release. Next planned release is 1.7.0.
